utils/vlog/tests/v2x_tests.cmake

```diff
@@ -118,9 +125,10 @@ function(V2X_TEST_PB_TYPE)
   set(SRC ${NAME}.sim.v)
   v2x(NAME ${NAME} SRCS ${SRC} TOP_MODULE ${TOP_MODULE})
 
-  v2x_test_generic(NAME ${NAME} TOP_MODULE ${TOP_MODULE} TYPE pb_type)
```

Comment:
I think you moved the arguments in the wrong direction.  NAME isn't the target name, TOP_MODULE is.  NAME implies control over the target name, which you have removed.  Your usage of DIFF is a good example of how it should work. NAME should be the target.  If the function is not going to respect NAME, it shouldn't have an argument named NAME.  Maybe REL_NAME or simply forgo NAME altogether, and just have TOP_MODULE in this instance.

Reply:
@kgugala